cocotb/regression.py

```diff
@@ -677,8 +677,17 @@ def add_option(self, name, optionlist):
         Args:
             name (str): Name of the option. Passed to test as a keyword
                 argument.
+            name (str, list, or tuple): A comma-separated string, list, or
+                tuple denoting option names. Passed to test as keyword
+                arguments.
 
             optionlist (list): A list of possible options for this test knob.
+                If N names were specified, this must be a list of N-tuples or
+                lists, where each element specifies a value for its respective
+                option.
+
+        .. versionchanged:: 1.5
+            Support groups of options
```

Comment:
I feel like this could be expanded to be a little clearer, but I'm not sure how to word it.

Reply:
Can you update the docstring at the top of this class to show how to use the new feature?

Reply:
Yes, I just updated the PR with that change